janis_unix/data_types/text.py

```diff
@@ -11,3 +15,43 @@ def name():
 
     def doc(self):
         return "A textfile, ending with .txt"
+
+    @classmethod
+    def basic_test(
+        cls,
+        tag: str,
+        min_size: int,
+        content: str,
+        line_count: int,
```

Comment:
I think we may want to make these two parameters optional too. As some of these information might not be available to the tester or may change on different runs.

Also, can we change `content` to `min_required_content` or something similar to indicate that we are not checking the exact content but just subcontent.